.github/workflows/ci-new.yml

```diff
@@ -257,11 +268,25 @@ jobs:
       # Keys can not be deleted, we can destroy key versions to avoid billing.
       #
       # See: https://cloud.google.com/kms/docs/faq#cannot_delete.
-      - name: 'Clean up kms key'
-        if: '${{ always()  && steps.create_kms_key.outcome == ''success'' }}'
+      - name: 'Clean up kms key when integ test failed'
```

Comment:
For the cleanups, I wonder if we can do in code, as this is not scalable if we have more than 1 key versions.

Reply:
Please see below comment.